### PR TITLE
Added OC and DAM implementation guides

### PR DESCRIPTION
**Pull request by markvanaalst**
## Description / Motivation
Adding implementation guides for OC and DAM

## How Has This Been Tested?
Local and Vercel

Pages changed:
- [DAM](https://developer-portal-git-content-2bb300-sitecoretechnicalmarketing.vercel.app/dam-and-content-operations/dam)
- [OC](https://developer-portal-git-content-2bb300-sitecoretechnicalmarketing.vercel.app/commerce/ordercloud#implementation-guides)

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [X] Documentation update (non-breaking change; modified files are limited to the `/data` directory or other markdown files)

## Checklist:
- [X] I have read the Contributing guide.
- [X] My code/comments/docs fully adhere to the Code of Conduct.
- [ ] My change is a code change.
- [X] My change is a documentation change and there are NO other updates required.
- [ ] My change has new or updated images which are stored in the `/public/images` folder that need to be migrated to Sitecore DAM

### CODE REVIEW

**Comment by markvanaalst** [comment]
Waiting on approval from Recai

**Comment by jst-cyr** [comment]
Approval from Recai received.

**Review by jst-cyr — APPROVED** [review]
Recai has approved and I did a quick test as well and everything looks good!

**Pull request merged**
